container_pipeline/workers/scanners/runner.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/python
+
+import docker
+import json
+import logging
+import os
+import shutil
+import subprocess
+import config
+import hashlib
+
+from Atomic import Atomic, mount
```

Comment:
`Atomic` is being used below but we're not doing anything with the `self.atomic_object` created using it. `mount` is not used at all. So, again copy-paste leftover. Unused imports.